inspirehep/modules/theme/jinja2filters.py

```diff
@@ -684,21 +684,19 @@ def format_date(datetext):
 @blueprint.app_template_filter()
 def find_collection_from_url(url):
     """Returns the collection based on the URL."""
-    if 'conferences' in url:
+    if 'cc=conferences' in url or '/conferences' in url:
```

Comment:
It has though some troubles:

If I just have 'jobs' in the URL it will think it's about Jobs. If I have 'cc=conferencesfoo'  it will think it's about conferences.